### PR TITLE
added sidebar

### PR DESCRIPTION
**Pull request by MeenuKumari28**
Added [sidebar](https://ui-git-sticky-toc-sidebar-fifthtry.vercel.app/scenarios/slides/logged-in-templates/)
signin

### CODE REVIEW

File: components/header.ftd [hunk 1]
```diff
@@ -8,6 +8,7 @@ module site-data: site-data
 module user-data: user-data
 optional string product:
 ds.gap margin: $ds.spaces.vertical-gap.zero
+optional boolean $sidebar: false
```
Comment (Arpita-Jaiswal) [inline components/header.ftd:11]:
```suggestion
boolean $sidebar: false
```

Comment (Arpita-Jaiswal) [reply components/header.ftd:11]: Create a new page `page-with-sidebar` instead of using `full-page`.

File: components/header.ftd [hunk 2]
```diff
@@ -20,13 +21,24 @@ margin: $header.margin
...
 spacing: $ds.spaces.horizontal-gap.zero
 align-content: center
-link: $header.site-data.homepage
 inset: $ds.spaces.inset-square.zero
 
+-- ftd.image:
+src: $assets.files.assets.slides.menu.svg
+width.fixed.px: 32
+border-radius.px: 4
+border-width.px: 1
```
Comment (amitu) [inline components/header.ftd:30]: Use `ds.border.thin` etc. We can not hard code values like this.

File: components/slides/page-with-sidebar.ftd [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+site-data: $page.site-data
+user-data: $page.user-data
+product: slides
+$sidebar: $page.sidebar
+
+-- ftd.row:
+width: fill-container
+background.solid: $inherited.colors.background.base
+
+-- ftd.column:
```
Comment (amitu) [inline components/slides/page-with-sidebar.ftd:39]: Use `ds.column` and `inset` etc.

File: components/slides/page-with-sidebar.ftd [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+color: $inherited.colors.text
+
+-- phosphor.light: gear
+size: 32
+
+-- end: ftd.column
+
+-- end: ftd.row
+
+-- ftd.column:
```
Comment (amitu) [inline components/slides/page-with-sidebar.ftd:99]: Every time you use `ftd.*` you write a comment about why you need it, and why `ds.*` does not have what you need.

**Comment by amitu** [comment]
@yashveer29 we can not use hamburger menu to show hide side bar. Click on hamburger menu on this page for example to see what end users expect. It must be the whole site's primary navigation. Use some other icon and in a place other than where hamburger menu is placed to indicate. 

image

**Comment by amitu** [comment]
Screenshot 2023-12-28 at 11 39 34 AM

Just remove the collapse button. 

The items are not laid out properly, focus on basics first instead of adding feature (of dubious utility).

**Comment by yashveer29** [comment]
Screenshot 2023-12-28 at 12 15 44 PM


Screenshot 2023-12-28 at 12 35 46 PM



@MeenuKumari28 please make the changes

**Comment by amitu** [comment]
@yashveer29, looks like you sent comment before the image was uploaded. When you drop an image, wait for it to finish uploading, you will see `` change into `` when it is done uploading.

**Comment by yashveer29** [comment]
Screenshot 2023-12-28 at 12 48 15 PM

@MeenuKumari28 Please check the Ui for the sidebar it need to be changed in all the screens

**Comment by yashveer29** [comment]
Screenshot 2023-12-28 at 12 43 17 PM

@MeenuKumari28 this was the image

File: FASTN.ftd [hunk 5]
```diff
@@ -22,6 +22,6 @@ provided-via: ui.fifthtry.com/ds
 -- fastn.auto-import: ui.fifthtry.com/assets
 -- fastn.auto-import: ui.fifthtry.com/ds
 
--- fastn.redirects:
+;;-- fastn.redirects:
 
-/onboarding/scenarios/onboarding-ishaan/ -> /scenarios/cards/onboarding-ishaan/
+;;/onboarding/scenarios/onboarding-ishaan/ -> /scenarios/cards/onboarding-ishaan/
```
Comment (amitu) [inline FASTN.ftd:27]: Why did we do this? Oh because we have changed `-- fastn.redirects:` to `-- fastn.url-mappings:`.

Comment (amitu) [reply FASTN.ftd:27]: Replace `-- fastn.redirects:` with `-- fastn.url-mappings:` and it should work.

**Pull request merged**